[2019-05-13 08:59:40,162] INFO Running in a "--no-prompt" mode (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:41,762] INFO Implicit acceptance of the license below:  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:41,762] INFO Confluent Software Evaluation License (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:41,762] INFO https://www.confluent.io/software-evaluation-license (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:41,762] INFO Downloading component Kafka Connect Kinesis 1.1.2, provided by Confluent, Inc. from Confluent Hub and installing into /Users/yeva/code/confluent_home/share/confluent-hub-components (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:41,769] INFO Implicit confirmation of the question: Do you want to uninstall existing version 1.1.2? (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:44,344] INFO Adding installation directory to plugin path in the following files: (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:44,345] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:44,395] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:44,396] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:44,397] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:44,399] INFO  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:44,399] INFO Completed (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:44,758] INFO Running in a "--no-prompt" mode (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:46,068] INFO Implicit acceptance of the license below:  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:46,068] INFO Confluent Community License (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:46,068] INFO http://www.confluent.io/confluent-community-license (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:46,068] INFO Downloading component Kafka Connect S3 5.2.1, provided by Confluent, Inc. from Confluent Hub and installing into /Users/yeva/code/confluent_home/share/confluent-hub-components (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 08:59:46,072] INFO Implicit confirmation of the question: Do you want to uninstall existing version 5.2.1? (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:00:00,314] INFO Adding installation directory to plugin path in the following files: (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:00:00,315] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:00:00,348] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:00:00,349] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:00:00,350] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:00:00,351] INFO  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:00:00,351] INFO Completed (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:03:28,515] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, always=false, sampleName=Ops, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:28,521] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, always=false, sampleName=Ops, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:28,521] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, always=false, sampleName=Ops, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:28,522] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 09:03:28,572] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 09:03:28,573] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 09:03:28,577] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:28,578] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:28,578] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:28,578] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:28,578] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:03:28,578] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:03:28,631] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 09:03:28,685] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 09:03:28,685] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 09:03:28,685] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 09:03:28,690] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:28,690] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:28,695] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:28,696] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:30,617] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:30,622] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:30,622] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:30,623] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 09:03:30,662] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 09:03:30,663] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 09:03:30,667] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:30,668] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:30,668] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:30,668] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:30,668] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:03:30,669] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:03:30,715] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 09:03:30,769] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 09:03:30,769] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 09:03:30,769] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 09:03:30,773] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:30,773] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:30,775] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:30,776] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:32,622] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:32,627] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:32,627] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:32,628] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 09:03:32,666] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 09:03:32,668] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 09:03:32,670] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:32,671] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:32,671] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:32,671] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:32,671] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:03:32,672] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:03:32,714] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 09:03:32,765] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 09:03:32,766] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 09:03:32,766] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 09:03:32,769] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:32,770] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:32,772] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:32,773] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:34,681] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:34,687] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:34,687] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:03:34,688] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 09:03:34,728] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 09:03:34,730] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 09:03:34,732] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:34,733] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:34,733] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:34,733] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:03:34,733] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:03:34,734] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:03:34,776] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 09:03:34,827] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 09:03:34,828] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 09:03:34,828] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 09:03:34,831] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:34,832] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:34,834] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:03:34,834] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:08:36,131] INFO Running in a "--no-prompt" mode (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:37,618] INFO Implicit acceptance of the license below:  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:37,621] INFO Confluent Software Evaluation License (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:37,621] INFO https://www.confluent.io/software-evaluation-license (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:37,622] INFO Downloading component Kafka Connect Kinesis 1.1.2, provided by Confluent, Inc. from Confluent Hub and installing into /Users/yeva/code/confluent_home/share/confluent-hub-components (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:37,625] INFO Implicit confirmation of the question: Do you want to uninstall existing version 1.1.2? (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:39,709] INFO Adding installation directory to plugin path in the following files: (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:39,709] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:39,755] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:39,756] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:39,757] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:39,758] INFO  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:39,759] INFO Completed (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:40,120] INFO Running in a "--no-prompt" mode (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:41,441] INFO Implicit acceptance of the license below:  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:41,443] INFO Confluent Community License (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:41,443] INFO http://www.confluent.io/confluent-community-license (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:41,443] INFO Downloading component Kafka Connect S3 5.2.1, provided by Confluent, Inc. from Confluent Hub and installing into /Users/yeva/code/confluent_home/share/confluent-hub-components (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:41,447] INFO Implicit confirmation of the question: Do you want to uninstall existing version 5.2.1? (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:55,022] INFO Adding installation directory to plugin path in the following files: (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:55,023] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:55,058] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:55,059] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:55,060] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:55,061] INFO  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:08:55,061] INFO Completed (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:12:07,889] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:12:07,893] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:12:07,894] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:12:07,894] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 09:12:07,935] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 09:12:07,936] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 09:12:07,939] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:12:07,940] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:12:07,940] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:12:07,940] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:12:07,940] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:12:07,941] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:12:07,991] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 09:12:08,043] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 09:12:08,043] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 09:12:08,044] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 09:12:08,048] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:12:08,049] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:12:08,051] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:12:08,052] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:12:09,895] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:12:09,899] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:12:09,899] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 09:12:09,900] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 09:12:09,937] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 09:12:09,938] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 09:12:09,941] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:12:09,942] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:12:09,942] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:12:09,942] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 09:12:09,942] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:12:09,942] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 09:12:09,983] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 09:12:10,037] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 09:12:10,037] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 09:12:10,037] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 09:12:10,041] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:12:10,042] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:12:10,044] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:12:10,044] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 09:56:51,838] INFO Running in a "--no-prompt" mode (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:53,330] INFO Implicit acceptance of the license below:  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:53,330] INFO Confluent Software Evaluation License (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:53,330] INFO https://www.confluent.io/software-evaluation-license (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:53,330] INFO Downloading component Kafka Connect Kinesis 1.1.2, provided by Confluent, Inc. from Confluent Hub and installing into /Users/yeva/code/confluent_home/share/confluent-hub-components (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:53,334] INFO Implicit confirmation of the question: Do you want to uninstall existing version 1.1.2? (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:55,443] INFO Adding installation directory to plugin path in the following files: (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:55,443] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:55,490] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:55,491] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:55,492] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:55,493] INFO  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:55,493] INFO Completed (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:55,842] INFO Running in a "--no-prompt" mode (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:57,372] INFO Implicit acceptance of the license below:  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:57,372] INFO Confluent Community License (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:57,372] INFO http://www.confluent.io/confluent-community-license (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:57,372] INFO Downloading component Kafka Connect S3 5.2.1, provided by Confluent, Inc. from Confluent Hub and installing into /Users/yeva/code/confluent_home/share/confluent-hub-components (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:56:57,376] INFO Implicit confirmation of the question: Do you want to uninstall existing version 5.2.1? (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:57:09,896] INFO Adding installation directory to plugin path in the following files: (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:57:09,896] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:57:09,933] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:57:09,934] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:57:09,935] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:57:09,935] INFO  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 09:57:09,935] INFO Completed (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:46,348] INFO Running in a "--no-prompt" mode (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:47,915] INFO Implicit acceptance of the license below:  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:47,915] INFO Confluent Software Evaluation License (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:47,915] INFO https://www.confluent.io/software-evaluation-license (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:47,916] INFO Downloading component Kafka Connect Kinesis 1.1.2, provided by Confluent, Inc. from Confluent Hub and installing into /Users/yeva/code/confluent_home/share/confluent-hub-components (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:47,919] INFO Implicit confirmation of the question: Do you want to uninstall existing version 1.1.2? (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:49,745] INFO Adding installation directory to plugin path in the following files: (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:49,746] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:49,792] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:49,793] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:49,795] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:49,796] INFO  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:49,796] INFO Completed (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:50,173] INFO Running in a "--no-prompt" mode (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:51,530] INFO Implicit acceptance of the license below:  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:51,530] INFO Confluent Community License (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:51,530] INFO http://www.confluent.io/confluent-community-license (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:51,531] INFO Downloading component Kafka Connect S3 5.2.1, provided by Confluent, Inc. from Confluent Hub and installing into /Users/yeva/code/confluent_home/share/confluent-hub-components (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:11:51,534] INFO Implicit confirmation of the question: Do you want to uninstall existing version 5.2.1? (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:12:04,333] INFO Adding installation directory to plugin path in the following files: (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:12:04,333] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:12:04,369] INFO   /Users/yeva/code/confluent_home/etc/kafka/connect-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:12:04,370] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-distributed.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:12:04,371] INFO   /Users/yeva/code/confluent_home/etc/schema-registry/connect-avro-standalone.properties (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:12:04,372] INFO  (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:12:04,372] INFO Completed (io.confluent.connect.hub.utils.IOUtils)
[2019-05-13 10:15:40,073] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:40,079] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:40,079] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:40,080] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 10:15:40,127] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 10:15:40,129] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 10:15:40,133] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:40,133] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:40,133] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:40,134] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:40,134] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 10:15:40,134] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 10:15:40,188] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 10:15:40,241] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 10:15:40,241] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 10:15:40,241] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 10:15:40,246] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:40,247] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:40,250] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:40,251] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:42,157] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:42,162] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:42,162] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(always=false, about=, sampleName=Ops, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:42,163] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 10:15:42,199] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 10:15:42,201] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 10:15:42,204] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:42,205] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:42,205] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:42,205] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:42,205] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 10:15:42,205] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 10:15:42,250] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 10:15:42,300] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 10:15:42,300] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 10:15:42,300] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 10:15:42,304] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:42,304] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:42,306] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:42,307] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:44,177] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:44,181] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:44,182] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:44,182] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 10:15:44,218] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 10:15:44,220] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 10:15:44,223] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:44,223] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:44,223] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:44,223] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:44,224] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 10:15:44,224] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 10:15:44,270] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 10:15:44,322] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 10:15:44,322] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 10:15:44,323] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 10:15:44,326] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:44,326] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:44,329] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:44,329] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:46,186] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginSuccess with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of successful kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:46,190] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.loginFailure with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[Rate of failed kerberos logins and latency (milliseconds)], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:46,191] DEBUG field org.apache.hadoop.metrics2.lib.MutableRate org.apache.hadoop.security.UserGroupInformation$UgiMetrics.getGroups with annotation @org.apache.hadoop.metrics2.annotation.Metric(about=, sampleName=Ops, always=false, type=DEFAULT, value=[GetGroups], valueName=Time) (org.apache.hadoop.metrics2.lib.MutableMetricsFactory)
[2019-05-13 10:15:46,191] DEBUG UgiMetrics, User and group related metrics (org.apache.hadoop.metrics2.impl.MetricsSystemImpl)
[2019-05-13 10:15:46,229] DEBUG Kerberos krb5 configuration not found, setting default realm to empty (org.apache.hadoop.security.authentication.util.KerberosName)
[2019-05-13 10:15:46,230] DEBUG  Creating new Groups object (org.apache.hadoop.security.Groups)
[2019-05-13 10:15:46,234] DEBUG Trying to load the custom-built native-hadoop library... (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:46,235] DEBUG Failed to load native-hadoop with error: java.lang.UnsatisfiedLinkError: no hadoop in java.library.path (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:46,235] DEBUG java.library.path=/Users/yeva/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:. (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:46,235] WARN Unable to load native-hadoop library for your platform... using builtin-java classes where applicable (org.apache.hadoop.util.NativeCodeLoader)
[2019-05-13 10:15:46,235] DEBUG Falling back to shell based (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 10:15:46,235] DEBUG Group mapping impl=org.apache.hadoop.security.ShellBasedUnixGroupsMapping (org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback)
[2019-05-13 10:15:46,281] DEBUG Failed to detect a valid hadoop home directory (org.apache.hadoop.util.Shell)
java.io.IOException: HADOOP_HOME or hadoop.home.dir are not set.
	at org.apache.hadoop.util.Shell.checkHadoopHome(Shell.java:302)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:327)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:257)
	at org.apache.hadoop.security.UserGroupInformation.ensureInitialized(UserGroupInformation.java:234)
	at org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject(UserGroupInformation.java:749)
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:734)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:607)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2748)
	at org.apache.hadoop.fs.FileSystem$Cache$Key.<init>(FileSystem.java:2740)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2606)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:368)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:167)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:352)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:296)
	at org.apache.avro.tool.Util.openFromFS(Util.java:88)
	at org.apache.avro.tool.Util.fileOrStdin(Util.java:60)
	at org.apache.avro.tool.DataFileReadTool.run(DataFileReadTool.java:68)
	at org.apache.avro.tool.Main.run(Main.java:87)
	at org.apache.avro.tool.Main.main(Main.java:76)
[2019-05-13 10:15:46,335] DEBUG setsid is not available on this machine. So not using it. (org.apache.hadoop.util.Shell)
[2019-05-13 10:15:46,335] DEBUG setsid exited with exit code 0 (org.apache.hadoop.util.Shell)
[2019-05-13 10:15:46,335] DEBUG Group mapping impl=org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback; cacheTimeout=300000; warningDeltaMs=5000 (org.apache.hadoop.security.Groups)
[2019-05-13 10:15:46,339] DEBUG hadoop login (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:46,339] DEBUG hadoop login commit (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:46,342] DEBUG using local user:UnixPrincipal: yeva (org.apache.hadoop.security.UserGroupInformation)
[2019-05-13 10:15:46,343] DEBUG UGI loginUser:yeva (auth:SIMPLE) (org.apache.hadoop.security.UserGroupInformation)
